php-nginx/php.ini

```diff
@@ -478,7 +478,7 @@ error_reporting = E_ALL & ~E_DEPRECATED & ~E_STRICT
 ; Development Value: On
 ; Production Value: Off
 ; http://php.net/display-errors
-display_errors = Off
+display_errors = stderr
```

Comment:
FYI Travis build is failing because phpinfo.php shows a warning like:
```
Warning: phpinfo() has been disabled for security reasons in /app/phpinfo.php on line 19
```